composer.json

```diff
@@ -40,7 +40,7 @@
       "phpstan/extension-installer": true
     },
     "platform": {
-      "php": "7.2"
+      "php": "7.3"
```

Comment:
Why is this necessary? The plugin requires PHP 7.2, which is still being tested. Doesn't this cause problems when trying to do `composer install`?

Reply:
We should be able to keep this at 7.2, `yoast/phpunit-polyfills` can then install whatever PHPUnit version that's supported on the current PHP version. See https://github.com/Yoast/PHPUnit-Polyfills